Library/ViewModels/DiscoveryViewModel.swift

```diff
@@ -126,6 +126,13 @@ public final class DiscoveryViewModel: DiscoveryViewModelType, DiscoveryViewMode
       .takeWhen(self.pageTransitionCompletedProperty.signal.filter(isTrue))
       .map { sorts[$0] }
 
+    let prevSortedPage = Signal.merge(
+      swipeToSort,
+      self.sortPagerSelectedSortProperty.signal.skipNil()
+    )
+    .combinePrevious(.magic)
```

Comment:
Could you explain the use of `.combinePrevious(.magic)`?

Reply:
`.combinePrevious(.magic)` uses scan to to get value of previously emitted and current event. I used `.magic` as the initial value because, on `Application Opened` previous sorted item should be equals current sorted, as the first sorted item is `.magic`.